tensorbay/cli/draft.py

```diff
@@ -64,20 +60,25 @@ def _list_drafts(dataset_client: DatasetClientType, info: TBRN) -> None:
 
     if info.is_draft:
         draft = dataset_client.get_draft(info.draft_number)
-        click.echo(f"Draft: {TBRN(info.dataset_name, draft_number=draft.number).get_tbrn()}")
-        _echo_draft(dataset_client, title=draft.title)
+        click.echo(f"Draft: {info.get_tbrn()}")
+        _echo_draft(dataset_client, title=draft.title, branch_name=draft.branch_name)
     else:
         for draft in dataset_client.list_drafts():
             click.echo(f"Draft: {TBRN(info.dataset_name, draft_number=draft.number).get_tbrn()}")
-            _echo_draft(dataset_client, title=draft.title)
+            _echo_draft(dataset_client, title=draft.title, branch_name=draft.branch_name)
+
 
+def _echo_draft(
+    dataset_client: DatasetClientType, title: str = "", branch_name: Optional[str] = None
+) -> None:
+    if not branch_name:
+        branch_name = dataset_client.status.branch_name
 
-def _echo_draft(dataset_client: DatasetClientType, title: str = "") -> None:
     try:
-        click.echo(f"Branch: main({dataset_client.get_commit('main').commit_id}) -> main")
+        click.echo(f"Branch: {branch_name}({dataset_client.get_commit(branch_name).commit_id})")
```

Comment:
check the behavior of `get_commit` when commit has no history
its behavior may changed in pylon v2.6